codecov.yml

```diff
@@ -0,0 +1 @@
+comment: off
```

Comment:
As a reviewer, I rely on these comments to see if the PR has tests. What's the reason to disable it?

Reply:
I find the Codecov comments distracting. Is there a way to configure visibility by user?